### PR TITLE
Calculate and display area of analysis shape

### PR DESCRIPTION
**Pull request by CloudNiner**
Always show area in sq mi
Add npm-debug.log to .gitignore
Uses area calculation functions from:
https://github.com/mapbox/geojson-area/blob/v0.2.1/index.js

### CODE REVIEW

**Comment by CloudNiner** [comment]
imls-area-1
imls-area-3
imls-area-4

File: app/scripts/views/museum/museum-controller.js [hunk 1]
```diff
@@ -228,6 +231,17 @@
             });
         }
 
+        function setACSArea(areaMeters) {
+            var areaMiles = Math.abs(areaMeters) * 0.000621371 * 0.000621371;
```
Comment (sharph) [inline app/scripts/views/museum/museum-controller.js]: Is that double multiplication correct?

Comment (CloudNiner) [inline app/scripts/views/museum/museum-controller.js]: Yup, converting from area to area so you have to multiply by the conversion factor for each dimension.

Comment (sharph) [inline app/scripts/views/museum/museum-controller.js]: Oh, yeah. Makes sense.

**Comment by sharph** [comment]
:+1:

**Pull request merged**